### ISSUE TITLE
ZCL_EXCEL_WORKSHEET~DELETE_MERGE logic need to fix

### ISSUE THREAD

**Issue by AtomKrieg** [abap2xlsx/abap2xlsx#511]
```
method delete_merge.

  field-symbols: <ls_merged_cell> like line of me->mt_merged_cells.
  data: is_merged type abap_bool.
  data: lv_column type i.

*--------------------------------------------------------------------*
* If cell information is passed delete merge including this cell,
* otherwise delete all merges
*--------------------------------------------------------------------*
  if   ip_cell_column is initial or ip_cell_row    is initial.
    clear me->mt_merged_cells.
  else.

*>>  this loop always delete first line or do nothing
*    loop at me->mt_merged_cells assigning <ls_merged_cell>.
*      is_merged = me->is_cell_merged( ip_column = ip_cell_column
*                                      ip_row    = ip_cell_row ).
*      IF is_merged = abap_true.
*        DELETE me->mt_merged_cells.  " Delete this merge, that includes this cell
*        EXIT.
*      ENDIF.
*    endloop.

*>>  the fix
    lv_column = zcl_excel_common=>convert_column2int( ip_cell_column ).

    loop at me->mt_merged_cells transporting no fields 
    where 
        ( col_from <= lv_column and col_to >= lv_column )
    and 
        ( row_from <= ip_cell_row and row_to >= ip_cell_row ).

      delete me->mt_merged_cells.
      exit.
    endloop.
*>>  the fix
  endif.


endmethod.
```

**Comment by gregorwolf** [comment]
Can you please send a pull request? Also it would be great if you can add a unit test that documents the issue.

**Comment by AtomKrieg** [comment]
https://github.com/AtomKrieg/abap2xlsx/pull/1

**Comment by larshp** [comment]
hi, the PR should end up in this repository, https://github.com/ivanfemia/abap2xlsx/pulls

see https://guides.github.com/introduction/flow/ and https://help.github.com/articles/creating-a-pull-request-from-a-fork/

**Comment by ivanfemia** [comment]
Fixed with commit #512
